app/routes/password/reset.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+const { Route, get, inject: { service } } = Ember;
+
+export default Route.extend({
+
+  ajax: service(),
+  session: service(),
+
```

Comment:
To get the same functionality you proposed by changing the route in API, you could do something like:

```Javascript
queryParams: {
  token: { refreshModel: true }
}
```

If you add that code into the route, the `params` hash in the model hook will contain a `token` key.

The standard way to specify these query parameters which are not part of the dynamic route URL would be to specify them in the controller:

```
export default Controller.extend({
  queryParams: ['token']
});
```

but the way I described above allows us to skip unnecessarily defining a controller file.